src/ansys/additive/core/parametric_study/display/ave_grain_size_plot.py

```diff
@@ -44,7 +44,7 @@
 
 
 def ave_grain_size_plot(ps: ParametricStudy):
-    """Generates a contour plot of build rate and relative density.
+    """Plots average grain size for laser power versus scan speed.
```

Comment:
```suggestion
    """Plot average grain size for laser power versus scan speed.
```
Descriptions for methods and functions start with verbs without "s" or "es" endings.